backend/infrahub/config.py

```diff
@@ -192,6 +192,7 @@ class DatabaseSettings(BaseSettings):
     password: str = "admin"
     address: str = "localhost"
     port: int = 7687
+    neo4j_http_port: int = 7474
```

Comment:
If it's just for testing purpose, its's best to not add it to the config file
in this case I think we're fine having the value hardcoded for the test

Reply:
Right, this value was actually never read